### ISSUE TITLE
`kind="descriptions"` attribute

### ISSUE THREAD

**Issue by imahdio** [imahdio/HTML-Essential-Training#8]
in [" Captions and subtitles" lesson](http://168.119.85.43/html-essential-training-4/07-Media/03-Captions%20and%20subtitles.mp4) of  HTML Essential Training instructor says:

>We could create a vtt file that has descriptive information about what's happening visually in the video, and use kind=descriptions. This gives people the option of playing a track that reads aloud descriptions of what you miss if you aren't looking at the video. For instance, the astronaut jumps off the ladder and bounces a bit as he hits the ground. This makes a video more accessible to the blind and the visually impaired. Kind=chapters gives us a way to provide a text file that lists the chapters in the video, giving users a way to jump from one section in the video to another.

[based this example code](https://codepen.io/ma400/pen/rNGwRNr) reply to the following questions:

1. why that subtitle be disappeared without any functionality when use kind="descriptions" !
2. seems there is same issue when using kind=chapters

plz share your solutions regarding above issues in here

**Comment by AlvinRemastered** [comment]
1. descriptions are not subtitles they are textual descriptions for when the video is not available or cant be seen.
2. chapters is the title of the video which is used for when looking through the media resource

**Comment by ghost** [comment]
> 1. descriptions are not subtitles they are textual descriptions for when the video is not available or cant be seen.

yes, thats why the subtitle be disappeared, if you want to use it as subtitles use kind="subtitles"

> 2. chapters is the title of the video which is used for when looking through the media resource

same with the first one

so, `track` tag has some kind values with different functions, check this
https://www.w3schools.com/tags/att_track_kind.asp

**Comment by imahdio** [comment]
tnx to @AlvinRemastered and @nurulaisyah61 for your kind replies
as I found , Although `kind="chapters"` attribute cannot be implemented in pure html but it's mandatory to create Video with WebVTT Chapters , like the following examples:
https://codepen.io/adrianparr/pen/EENrgy/
http://thenewcode.com/977/Create-Interactive-HTML5-Video-with-WebVTT-Chapters

but I'm still looking for an example to see the functionality of using [descriptions attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track)
plz suggest some resources to show its funcationality in an interactive environment

**Comment by ghost** [comment]
https://act-rules.github.io/rules/ac7dc6

I found this... but not really can tell the difference haha... still searching.. will tell you if I find the good show for this issue

**Comment by MBEDOBE** [comment]
@nurulaisyah61  i expected the descriptions to read out the actions in the video but it's rather doing the opposite, just providing the audio of the video.

**Comment by ghost** [comment]
hmm, what do you mean @MBEDOBE ? do you hear the audio from the video by track kind="description" ?

**Comment by MBEDOBE** [comment]
> hmm, what do you mean @MBEDOBE ? do you hear the audio from the video by track kind="description" ?

yes, that's exactly what happens.

**Comment by ghost** [comment]
is the audio really reading the track of kind="descriptions" ?

I think it should be the audio of the track because the descriptions kind of the track should be helping the blind people... but I cant hear the audio of the track at all.. can you give the video of it @MBEDOBE ?

**Comment by MBEDOBE** [comment]
> is the audio really reading the track of kind="descriptions" ?
> 
> I think it should be the audio of the track because the descriptions kind of the track should be helping the blind people... but I cant hear the audio of the track at all.. can you give the video of it @MBEDOBE ?

@nurulaisyah61 this is a repo of what i tested [https://github.com/MBEDOBE/test](https://github.com/MBEDOBE/test) , you can try it let's see the results because i still hear the audio from the video when kind="descriptions"

**Comment by imahdio** [comment]
tnx @MBEDOBE  for leaving your reply on this issue
but it seems this issue couldnot reach into any result because of the following quotation based the @nurulaisyah61 [resource](https://act-rules.github.io/rules/ac7dc6)
> Currently the description track is not supported by most assistive technologies. Video players may be able to work around the lack of support for the description track by using aria-live but few do this today.

BTW highly appriciate for any other opinion if there be any
